WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -905,13 +906,13 @@ class AztecPostViewController: UIViewController, PostEditor {
         setHTML(content)
     }
 
-    func reloadBlogPickerButton() {
+    func reloadBlogTitleView() {
         var pickerTitle = post.blog.url ?? String()
         if let blogName = post.blog.settings?.name, blogName.isEmpty == false {
             pickerTitle = blogName
         }
 
-        navigationBarManager.reloadBlogPickerButton(with: pickerTitle, enabled: !isSingleSiteMode)
+        navigationBarManager.reloadBlogTitleView(text: pickerTitle)
```

Comment:
Maybe we can rename `pickerTitle` as well?

Reply:
Fixed at 33aaa72